### PR TITLE
fix manifest

### PR DESCRIPTION
**Pull request by scgbckbone**

### CODE REVIEW

File: ngu/ngu_tests/test_random.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 import ngu
 
 for trial in range(100):
-    v = [ngu.random.uint32() for i in range(1000)]
+    v = [ngu.random.uint32() for i in range(100)]
```
Comment (scgbckbone) [inline ngu/ngu_tests/test_random.py:5]: tooo slow on stm32

File: ngu/manifest.py [hunk 2]
```diff
@@ -3,15 +3,16 @@
 ], opt=3)
 
 import os
-if not os.environ.get('EXCLUDE_NGU_TESTS', False):
+if not int(os.environ.get('EXCLUDE_NGU_TESTS', 0)):
```
Comment (scgbckbone) [inline ngu/manifest.py:6]: sending `EXCLUDE_NGU_TESTS=0` in terminal, causes tests to be not included, because '0' is truthy value. Only way to include tests currently is with `EXCLUDE_NGU_TESTS=` (empty)

File: ngu/manifest.py [hunk 3]
```diff
@@ -3,15 +3,16 @@
...
     print("NGU: including tests")
     freeze_as_mpy('', [
         "ngu_tests/run.py",
         "ngu_tests/b39_data.py",
         "ngu_tests/b39_vectors.py",
         "ngu_tests/b340_vectors.py",
-        "ngu_tests/test_bip39.py",
         "ngu_tests/test_aes_gen.py",
+        "ngu_tests/test_bip32.py",
```
Comment (scgbckbone) [inline ngu/manifest.py:14]: failed to add test_bip32.py to manifest when added to repo
